recipes/aiozmq/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "aiozmq" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b41d589a715acc0d42b525ef7084bbcdc03c986ef5a90e212ee7f086125cf6c0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - pyzmq >=13.1
+  run:
+    - python
+    - pyzmq >=13.1
+
+test:
+  imports:
+    - aiozmq
+    - aiozmq.cli
+    - aiozmq.rpc
+  commands:
+    - aiozmq-proxy --help
+  requires:
+    - msgpack-python >=0.4.0
+    - pyzmq >=13.1
+
+about:
+  home: http://aiozmq.readthedocs.org
+  license: BSD
```

Comment:
This should specify the variant of the BSD licence, likely `BSD-2-Clause` or `BSD-3-Clause`.